recipes/r-flowml/meta.yaml

```diff
@@ -0,0 +1,125 @@
+{% set version = '0.1.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-flowml
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/flowml_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/flowml/flowml_{{ version }}.tar.gz
+  sha256: 1bfc1d22dc841b44eb7e0d4202387ae349803a723351f8f2dd8b7c67407f5abb
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-abcanalysis
+    - r-r6
+    - r-caret
+    - r-data.table
+    - r-dplyr
+    - r-fastshap
+    - r-furrr
+    - r-future
+    - r-magrittr
+    - r-optparse
+    - r-purrr
+    - r-readr
+    - r-rjson
+    - r-rlang
+    - r-rsample
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-vip
+  run:
+    - r-base
+    - r-abcanalysis
+    - r-r6
+    - r-caret
+    - r-data.table
+    - r-dplyr
+    - r-fastshap
+    - r-furrr
+    - r-future
+    - r-magrittr
+    - r-optparse
+    - r-purrr
+    - r-readr
+    - r-rjson
+    - r-rlang
+    - r-rsample
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-vip
+
+test:
+  commands:
+    - $R -e "library('flowml')"           # [not win]
+    - "\"%R%\" -e \"library('flowml')\""  # [win]
+
+about:
+  home: https://github.com/Boehringer-Ingelheim/flowml
+  license: GPL-3
```

Comment:
```suggestion
  license: GPL-3.0-or-later
```

Reply:
I will do so, once I see the r-abcanalysis package available on the package index. The r-fastshap one is there now 🥳